### DISCUSSION TITLE
Start menu close animation nonexistent (OS: Windows 11, Style: Windows 11)

### DISCUSSION THREAD

**Discussion by Destructor-Ben** [valinet/ExplorerPatcher#2322] [Q&A] [answered]
Essentially, the start menu has no close animation. I took a screen recording of it, and it appears to only have one frame where the animation runs and it is very subtle.

Weird behaviour with the search menu when it is opened as well (see video, I believe the search menu has an exit animation but not the start menu).

Video:
https://github.com/valinet/ExplorerPatcher/assets/62684035/ff625a92-a83d-4179-b216-692205318a2b

**Comment by pyrates999** [comment, marked as answer]
Please close this.

Duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1296
